blob/gcsblob/gcsblob.go

```diff
@@ -253,8 +253,13 @@ func (b *bucket) ErrorCode(err error) gcerrors.ErrorCode {
 	if err == storage.ErrObjectNotExist {
 		return gcerrors.NotFound
 	}
-	if gerr, ok := err.(*googleapi.Error); ok && gerr.Code == 404 {
-		return gcerrors.NotFound
+	if gerr, ok := err.(*googleapi.Error); ok {
+		switch gerr.Code {
+		case 404:
```

Comment:
nit: You can use the `http.Status*` constants, like `http.StatusNotFound`, to make this mapping a little easier to follow.

Reply:
Done.